### DISCUSSION TITLE
possible? -> use jit but include all "w-X" classes doesn´t matter if used or not

### DISCUSSION THREAD

**Discussion by michabbb** [tailwindlabs/tailwindcss#8274] [Help] [answered]
hi there,

i am using jit and tailwind is scanning all my blade files for classes and includes them, so far so good.
but if a class gets set dynamically like `w-$width` tailwind of course is unable to include "w-10" into my CSS.
so that´s why I wanted to ask, if it´s possible to tell the precompiled to include all "w-X" classes into my CSS,
doesn´t matter if it´s used somewhere or not. otherwise I always have to use a fake-blade-file with a fake `<div>`
where I put in `<div class="w-1 w-2 w-3 w-4....` which is stupid, of course.

thanks for any help!

**Comment by Lelectrolux** [comment, marked as answer]
Yes, it can be done. Read https://tailwindcss.com/docs/content-configuration#safelisting-classes. Don't forget the `Using regular expressions` paragraph.

```js
module.exports = {
  content: [ ... ],
  safelist: [
    { pattern: /w-.+/ },
  ],
  // ...
}
```

**Reply by michabbb** [reply]
amazing, exactly what i was looking for, thanks 👍
